### DISCUSSION TITLE
Terraform reports "The GitHub user that published this artifact no longer exists. See the FAQ for more information"

### DISCUSSION THREAD

**Discussion by jlpedrosa** [terraform-routeros/terraform-provider-routeros#392] [General]
Hi @vaerh  it looks like terraform artifactory report seems to be linked to the old owner of the provider, I think it should be updated.
I think it created some questions on the community:
https://www.reddit.com/r/mikrotik/comments/1bltz42/comment/kw7j33e/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

![image](https://github.com/terraform-routeros/terraform-provider-routeros/assets/4932947/ae49375d-e5d1-4314-b538-08f39da72329)

**Comment by vaerh** [comment]
Yes, I've seen the discussion, but I can't do anything about it yet. The registry is hardwired into this organization and I don't have access to it. I would like to see a response from HC with a description of the steps, then I can ask the author for help if needed.
If I understand correctly what's going on here. 
the automaton that publishes release links on GitHub didn't see artifacts for version 1.35 and that's where the problems started. It thinks the user may have been deleted because the artifacts are not available.
